bootstrap/internal/controllers/rke2config_controller.go

```diff
@@ -341,6 +372,39 @@ func (r *RKE2ConfigReconciler) handleClusterNotInitialized(ctx context.Context,
 	return ctrl.Result{}, nil
 }
 
+func (r *RKE2ConfigReconciler) generateFileListIncludingRegistries(scope *Scope, ctx context.Context, configFiles []bootstrapv1.File) ([]bootstrapv1.File, error) {
+	registries, registryFiles, err := rke2.GenerateRegistries(rke2.RKE2ConfigRegistry{
+		Registry: scope.Config.Spec.PrivateRegistriesConfig,
+		Client:   r.Client,
+		Ctx:      ctx,
+		Logger:   scope.Logger,
+	})
+
+	if err != nil {
+		scope.Logger.Error(err, "unable to generate registries.yaml for Init Control Plane node")
+		return nil, err
+	}
+
+	registriesYAML, err := kubeyaml.Marshal(registries)
+	if err != nil {
+		scope.Logger.Error(err, "unable to marshall registries.yaml")
+		return nil, err
+	}
+	scope.Logger.Info("Registries.yaml marshalled successfully")
```

Comment:
Feels like a debug message as opposed to a standard into. Maybe verbosity 2 or even 4

Reply:
Put verbosity 4